src/app/manage-users/manage-user-details/manage-user-details.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-manage-user-details',
+  templateUrl: './manage-user-details.component.html',
+  styleUrls: ['./manage-user-details.component.css']
+})
+export class ManageUserDetailsComponent implements OnInit {
```

Comment:
If `OnInit` is not used, then it can removed

Reply:
Removed the OnInit